heclib/javaHeclib/src/Hec_ztsStore.c

```diff
@@ -244,20 +244,20 @@ JNIEXPORT jint JNICALL Java_hec_heclib_util_Heclib_Hec_1ztsStore(
 		number = (int)(*env)->GetIntField(env, j_timeSeriesContainer, fid);
 		tss->timeGranularitySeconds = number;
 	}
-	if (tss->timeGranularitySeconds == 0) tss->timeGranularitySeconds = 60;
+	if (tss->timeGranularitySeconds == 0) tss->timeGranularitySeconds = SECS_IN_1_MINUTE;
 	
 	if (startTime != 0) {
-		if (tss->timeGranularitySeconds == 60) {
+		if (tss->timeGranularitySeconds == SECS_IN_1_MINUTE) {
```

Comment:
I think this should be MINUTE_GRANULARITY instead of SECS_IN_1_MINUTE

Reply:
Done. There were an embarrassing number of these!